include/uapi/linux/v4l2-controls.h

```diff
@@ -192,9 +192,15 @@ enum v4l2_colorfx {
  * We reserve 16 controls for this driver. */
 #define V4L2_CID_USER_IMX_BASE			(V4L2_CID_USER_BASE + 0x10b0)
 
+/*
+ * The base for the atmel isc driver controls.
+ * We reserve 32 controls for this driver.
+ */
+#define V4L2_CID_USER_ATMEL_ISC_BASE           (V4L2_CID_USER_BASE + 0x10c0)
```

Comment:
Spaces used in padding instead of TABs (which the other lines use).

Reply:
That's what I get for copy/pasting from the commit rather than loading the source file into the editor. Fixed.